vtkext/private/module/vtkF3DConfigure.h.in

```diff
@@ -3,7 +3,9 @@
 
 #include <string>
 
-static const std::string F3D_RESERVED_STRING = "f3d_reserved";
+// TODO REMOVE THIS STUFF!
```

Comment:
I'm not sure to understand all the changes about the reserved keyword, but I assume it's not a big deal if you are planning to remove it.

Reply:
yup